pipeline-runner/R/qc-5-filter_doublets.R

```diff
@@ -17,47 +17,47 @@
 #' @export
 #' @return a list with the filtered seurat object by doublet score, the config and the plot values
 #'
-filter_doublets <- function(scdata, config, sample_id, cells_id, task_name = "doubletScores", num_cells_to_downsample = 6000) {
-  cells_id.sample <- cells_id[[sample_id]]
+filter_doublets <- function(scdata_list, config, sample_id, cells_id, task_name = "doubletScores", num_cells_to_downsample = 6000) {
+  sample_cell_ids <- cells_id[[sample_id]]
 
-  if (length(cells_id.sample) == 0) {
-    return(list(data = scdata, new_ids = cells_id, config = config, plotData = list()))
+  if (length(sample_cell_ids) == 0) {
+    return(list(data = scdata_list[[sample_id]], new_ids = cells_id, config = config, plotData = list()))
   }
 
-  scdata.sample <- subset_ids(scdata, cells_id.sample)
+  sample_data <- subset_ids(scdata_list[[sample_id]], sample_cell_ids)
 
   # Check if the experiment has doubletScores
-  if (!"doublet_scores" %in% colnames(scdata@meta.data)) {
+  if (!"doublet_scores" %in% colnames(scdata_list[[sample_id]]@meta.data)) {
     message("Warning! No doubletScores scores has been computed for this experiment!")
     guidata <- list()
-    return(list(data = scdata, config = config, plotData = guidata))
+    return(list(data = scdata_list[[sample_id]], config = config, plotData = guidata))
   }
 
-  probabilityThreshold <- config$filterSettings[["probabilityThreshold"]]
+  probability_threshold <- config$filterSettings[["probabilityThreshold"]]
 
   # Check if it is required to compute sensible values. From the function 'generate_default_values_doubletScores', it is expected
-  # to get a value --> probabilityThreshold.
+  # to get a value --> probability_threshold.
```

Comment:
How about you change comment `to get a value --> probability_threshold` to `to get a probability threshold value`, so that the validity of the comment doesn't depend on the naming you use for the probability threshold value?